README.md

```diff
@@ -1,44 +1,71 @@
+# GSP [![IRC](https://img.shields.io/badge/kubernetes-v1.12.6-0099ef.svg)]() <img align="right" src="./docs/assets/gsp.png" alt="gsp" width="200" height="whatever">
 
-# What is the GDS Supported Platform?
+GSP is a Kubernetes distribution maintained by the [Govenement Digital Service](https://www.gov.uk/government/organisations/government-digital-service) created to meet the common needs of running digital services in production.
```

Comment:
```suggestion
The GDS Supported Platform (GSP) is a Kubernetes distribution maintained by the [Govenement Digital Service](https://www.gov.uk/government/organisations/government-digital-service) created to meet the common needs of running digital services in production.
```

Reply:
Damn was hoping nobody had noticed I stopped calling it that hehe... Are we _seriously_ sticking with an initialism that has another initialism in it heh ... I won't start a naming bikeshed here and I will conceed to mention it's backryomned name as you suguest ... But I do think something less abrasive to PaaS is required (they are a gds supported platform too!)

Reply:
expanded :done: